README.md

```diff
@@ -3,7 +3,7 @@
 [![hackmd-github-sync-badge](https://hackmd.io/N1cWyDM2S9eaAQtSvS0J_A/badge)](https://hackmd.io/N1cWyDM2S9eaAQtSvS0J_A?both)
 
 - **Title:** Machine Learning Model Extension
-- **Identifier:** [https://crim-ca.github.io/mlm-extension/v1.2.0/schema.json](https://crim-ca.github.io/mlm-extension/v1.2.0/schema.json)
+- **Identifier:** <https://crim-ca.github.io/mlm-extension/v1.2.0/schema.json>
```

Comment:
I don't think we should modify this. STAC extensions GitHub pages parses this information to generate the table in https://stac-extensions.github.io/.

Reply:
hmmm I was getting linting errors about using angle brackets and checked the processing extension, they use angle brackets and seem to get parsed ok https://raw.githubusercontent.com/stac-extensions/processing/main/README.md

but I checked again and I don't get the lint error (not sure why) so will revert.